app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,7 @@
             android:label="@string/title_activity_record"
             android:theme="@style/Theme.WorkoutApp" />
         <activity
-            android:name=".WorkoutActivity"
+            android:name=".MainActivity"
```

Comment:
I think it makes the most sense to start the app from your main activity

Reply:
Android says that the first screen that the users sees should be the screen that they want to access, 80% of the time.
Do you know that MainActivity is that screen?  Is it even a screen used by the application?
I urge caution on changing a flow that is this significant to the application, unless there's good reason for the change.

Reply:
I couldn't tell. To me it seemed like main activity was the home page which I would assume is where majority of the time spent would be.